RELEASE.md

```diff
@@ -0,0 +1,44 @@
+# Release procedure
+
+This is a manual release procedure instruction/checklist.
+
+
+## Prepare
+
+1. Update README's javadoc link with version (see 'site release', below)
+
+
+## Release
+
+1. `mvn -Prelease release:clean release:prepare release:perform`
+1. Promote manually to Maven Central with <https://oss.sonatype.org/>
+1. `git push --follow-tags`
+
+
+### Site release
+
+```sh
+cd target/checkout
+mvn site
+git checkout gh-pages
+mv target/site v/<version>
```

Comment:
1. Highlight the version information as it is mentioned in other steps.
1. Extract version to env variable.